### ISSUE TITLE
For the distillation process, do we absolutely need the rrna and trna path?

### ISSUE THREAD

**Issue by bellasmith121995** [shafferm/DRAM#71]
For the distillation process, do we absolutely need the rrna and trna path?

**Comment by shafferm** [comment]
Nope. You can leave them out and the only changes are that you won't get the tabs for tRNAs and rRNAs in your `metabolism_summary.xlsx` file and you won't get tRNA counts and rRNA locations in your `genome_stats.tsv`. It will not affect the `product.html` or `product.tsv` files.
